app/views/content/a-day-in-the-life-of-a-teacher.md

```diff
@@ -17,79 +18,145 @@
     - everyday
     - timetable
   quote:
-    q-sarah-primary:
-      text: "I like to get in early so I can get organised — in teaching, preparation is everything."
-      name: "Sarah, primary school teacher"
-      classes: quote--indent
-    q-sarah-geography-1:
-      text: "I have four free periods a week, which I use as my own time."
-      name: "Sarah, geography teacher"
-      classes: quote--indent
-    q-gavin:
-      text: "Teaching and learning does not stop with the bell at the end of the school day."
+    q-gavin-ict:
+      text: "Getting involved with extracurricular activities can be extremely rewarding."
       name: "Gavin, assistant headteacher"
       classes: quote--indent
-    q-sarah-geography-2:
+    q-jane:
+      text: "It’s all about the relationships you build with pupils. If they know they’ll ‘get away’ with disruptive learning, they will disrupt learning!"
+      name: "Jane, teaching adviser"
+      classes: quote--indent
+    q-sarah-geography:
       text: "I sometimes do a couple of hours at the weekend but I try to keep my weekends free to make sure I have some real downtime."
       name: "Sarah, geography teacher"
       classes: quote--indent
+    q-helen-food:
+      text: "It's not an easy career. There will be challenges along the way but persevere and the rewards are amazing. It's my 35th year and every day is still different. Do not expect too much of yourself too soon - 1 day at a time but you will make a difference and you will be remembered."
+      name: "Helen, food tech teacher"
+      classes: quote--indent
+    q-phil-secondary:
+      text: "Seeing the students arrive and knowing they can’t wait to get to my class. There’s no feeling like it."
+      name: "Phil, secondary teacher"
+      classes: quote--indent
+    q-jessica-science:
+      text: "There’s nothing better than those light bulb moments when students just get it."
+      name: "Jessica, head of science"
+      classes: quote--indent
+    q-richard-art:
+      text: "The fact that I continue to love the subject I teach has meant that I have never grown weary of doing what I do."
+      name: "Richard, art teacher"
+      classes: quote--indent
+    q-mike-secondary:
+      text: "If you’ve had a hard day, there’s always someone there to listen - even if it’s just a two-minute natter or moan."
+      name: "Mike, secondary teacher"
+      classes: quote--indent
+    q-aaron-science:
+      text: "Teaching offers you something few other careers can offer you. To make a real difference to a large number of students lives for the better."
+      name: "Aaron, science teacher"
+      classes: quote--indent
+
+
 ---
+Teaching is a challenging but rewarding career that allows you to make a significant impact on pupils' lives. With no two days being the same, it’s an excellent choice for those who thrive in a dynamic and varied work environment.
+
+$q-richard-art$
+
+## A day in the life of a teacher
+
```

Comment:
We say above that no day is the same - is it worth adding a note that this is an example of what teaching could be like?